docs/hpc/03_storage/03_data_transfers.md

```diff
@@ -1 +1,75 @@
-# Data transfers
+# Data Transfers
+
+## Introduction
+The main tools to transfer data to/from HPC systems
+
+-   Linux tools like scp and rsync
+-   Please use Data transfer nodes
+:::note
```

Comment:
Maybe convert this to a warning and let them know that excessive use of resources is discouraged.

Reply:
Yes, absolutely.  We really don't want people doing this, so it should be a warning.  Thanks!